HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -201,6 +216,13 @@ def self.apply_air_source_hvac_systems(model, cooling_system, heating_system,
 
     apply_installation_quality(model, heating_system, cooling_system, air_loop_unitary, htg_coil, clg_coil, control_zone)
 
+    if not max_pow_ratio_sch.nil?
+      # Only apply heating control when it's a heat pump
+      htg_coil = nil unless is_heatpump
+      htg_supp_coil = nil unless is_heatpump
+      apply_max_power_EMS(model, max_pow_ratio_sch, air_loop_unitary, control_zone, htg_supp_coil, clg_coil, htg_coil)
+    end
```

Comment:
It'd be nice to clean up the code because the business logic is spread out a bit. Right now we get the schedule and check for an error at the top of the method, provide a warning in the middle of the method, and apply the EMS at the end of the method. Can we just have a single `apply_max_power` method that is self contained and does all of this?